service/src/main/java/bio/terra/pipelines/service/JobsService.java

```diff
@@ -0,0 +1,68 @@
+package bio.terra.pipelines.service;
+
+import bio.terra.pipelines.db.JobsDao;
+import bio.terra.pipelines.service.model.Job;
+import bio.terra.pipelines.service.model.JobRequest;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/** The Jobs Service manages job requests to run the service's Scientific Pipelines. */
+@Component
+public class JobsService {
+
+  private static final Logger logger = LoggerFactory.getLogger(JobsService.class);
+
+  private final JobsDao jobsDao;
+
+  @Autowired
+  public JobsService(JobsDao jobsDao) {
+    this.jobsDao = jobsDao;
+  }
+
+  public UUID createJob(String userId, JobRequest jobRequest) {
+    UUID jobId = createJobId();
+    Timestamp timeSubmitted = getCurrentTimestamp();
```

Comment:
add TODO: in future get the timestamp from CBAS/Cromwell, don't generate it here

Reply:
actually, we want to keep this - this is the time we received the request and started the full process (of which the CBAS submission is only a part)